app/views/results/marker/_marker_panes.html.erb

```diff
@@ -29,13 +29,15 @@
                        locals: { files: files,
                                  can_download: true } %>
 
-            <button id='new_annotation_button' onclick='make_new_annotation(); return false;'>
-              <%= t('marker.new_annot')%>
-            </button>
-
-            <%= render partial: 'annotations/annotation_categories',
-                       locals: { annotation_categories: annotation_categories,
-                                 result_id: result.id } %>
+            <% if !@result.is_a_review? %>
+              <button id='new_annotation_button' onclick='make_new_annotation(); return false;'>
```

Comment:
1. Use `unless`.
2. I don't know why this button is not show to peer reviewers.